vpdq/cpp/CMakeLists.txt

```diff
@@ -63,18 +63,28 @@ add_executable(vpdq-hash-video
     bin/vpdq-hash-video.cpp
 )
 
+
 target_link_libraries(vpdqlib
     PkgConfig::LIBAV
 )
 
-target_link_libraries(vpdq-hash-video
+target_link_libraries(match-hashes-byline
     vpdqlib
+    PkgConfig::LIBAV
```

Comment:
Fairly certain you don't need PkgConfig::LIBAV because vpdqlib already has it, but from what I read it's better to specify this.

Reply:
Outside of my area of knowledge, I'm going to trust you :P